src/main/scala/org/ua/ilm/datamining/crawlers/youtube/ThreadCrawlerChannels.scala

```diff
@@ -0,0 +1,22 @@
+package org.ua.ilm.datamining.crawlers.youtube
+
+import java.util.concurrent.{Callable, Executors}
+
+import com.google.api.services.youtube.model.Channel
+
+class ThreadCrawlerChannels(key: String, arrIdChannels: Array[String],
```

Comment:
extend your class with non-parallel one and use father's methods in your requests. Also, than your naming for crawling methods may be the same. Mode (parallel or not) will be get from class name (YoutubeParallelCrawler.getChannels, YoutubeCrawler.getChannels for example)

Reply:
Same for parser